common/components/PrintableWallet/index.tsx

```diff
@@ -27,7 +28,7 @@ const print = (address: string, privateKey: string) => () =>
 
 const PrintableWallet: React.SFC<{ wallet: IFullWallet }> = ({ wallet }) => {
   const address = wallet.getAddressString();
-  const privateKey = wallet.getPrivateKeyString();
+  const privateKey = stripHexPrefix(wallet.getPrivateKeyString());
```

Comment:
Prefer stripping hex prefer at wallet method

Reply:
We'd have to wrap the library itself if we did this

Reply:
Agreed. I don't expect breaking changes here, so I don't think we should
have any trouble wrapping.

@HenryNguyen5 Perhaps we should introduce a separate PR for this, as I want to ensure that we design the wrapped lib correctly / with an independent design process from this PR.

Reply:
We have a bunch of utility functions dealing with `stripHex` `sanitizeHex` `stripHexPrefixAndLower` etc. I think having a consistent set of methods would be ideal, unless we really need all these different types of sanitization/mutation methods.